idaes/core/util/tests/test_phase_equilibria.py

```diff
@@ -0,0 +1,285 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+This module contains phase equilibria utility functions for use in IDAES models.
+"""
+
+import pytest
+
+import idaes.logger as idaeslog
+
+from pyomo.environ import ConcreteModel, Expression, Set, Block, Var
+# Import Pyomo units
+from pyomo.environ import units as pyunits
+
+from idaes.core import LiquidPhase, VaporPhase, Component
+from idaes.core.phases import PhaseType as PT
+
+from idaes.generic_models.properties.core.state_definitions import FTPx
+from idaes.generic_models.properties.core.eos.ideal import Ideal
+from idaes.generic_models.properties.core.eos.ceos import Cubic, CubicType
+from idaes.generic_models.properties.core.phase_equil import smooth_VLE
+from idaes.generic_models.properties.core.phase_equil.bubble_dew import \
+        IdealBubbleDew, LogBubbleDew
+from idaes.generic_models.properties.core.phase_equil.forms import fugacity, log_fugacity
+
+import idaes.generic_models.properties.core.pure.NIST as NIST
+import idaes.generic_models.properties.core.pure.RPP4 as RPP4
+import idaes.generic_models.properties.core.pure.RPP5 as RPP5
+
+from idaes.core import FlowsheetBlock
+
+from idaes.generic_models.properties.core.generic.generic_property import (
+        GenericParameterBlock)
+
+from idaes.core.util.phase_equilibria import (TXYDataClass,Txy_data)
+
+# Author: Alejandro Garciadiego
+@pytest.mark.unit
```

Comment:
As all of these tests involve a solver, they should be marked as `component` tests instead of `unit` tests. Given the nature of this tool, I am not sure how we would write `unit` tests as this tool inherently requires a solver.